model/config.go

```diff
@@ -295,7 +295,7 @@ type ServiceSettings struct {
 	DisableBotsWhenOwnerIsDeactivated                 *bool `restricted:"true"`
 }
 
-func (s *ServiceSettings) SetDefaults() {
+func (s *ServiceSettings) SetDefaults(isNew bool) {
```

Comment:
Adding `isNew` is /slightly/ annoying given that some methods take it and some don't. Propose instead adding an `isNew` flag on the config struct (not persisted/exported), and we set that in the top-level `SetDefaults` method.

Reply:
I agree, BUT: adding a field in Config struct will not help the child SetDefaults methods like:
```
func (s *SqlSettings) SetDefaults() {
```
since `SqlSettings` doesn't have a reference to top-level `Config`

Reply:
🤦‍♂️, you're right! Not sure what I was thinking there ;P